.storybook/stories/LendHomepageLoanCard.stories.js

```diff
@@ -0,0 +1,77 @@
+import Vue from 'vue'
+import StoryRouter from 'storybook-vue-router';
+import { text, number, boolean } from '@storybook/addon-knobs';
+import apolloStoryMixin from '../mixins/apollo-story-mixin';
+import loanDataPropStoryMixin from '../mixins/loan-data-prop-story-mixin';
+
+
+import LendHomepageLoanCard from '@/components/LoanCards/LendHomepageLoanCard';
+
+// import plugins
+import kivaPlugins from '@/plugins';
+Vue.use(kivaPlugins)
+
+export default {
+	title: 'Loan Cards/Lend Homepage Loan Card',
+	component: LendHomepageLoanCard,
+	decorators: [StoryRouter()],
+};
+
+export const Default = () => ({
+	mixins: [apolloStoryMixin, loanDataPropStoryMixin],
+	components: {
+		LendHomepageLoanCard,
+	},
+	props: {
+		amountLeft: {
+			type: Number,
+			default: number("Amount Left", 250),
```

Comment:
Could these props be part of loan-data-prop-story-mixin.js?

Reply:
Maybe, I think this gives us a little more flexibility. For example, grid loan card takes a title prop, the lend homepage card doesn't. While its a safe bet that they will probably all have props like "amountLeft" maybe they won't